docs/schemas/index.md

```diff
@@ -1,4 +1,4 @@
-# Schemas
+# Geoscience object schemas
```

Comment:
I thought we called these "Evo schemas" these days?

Reply:
Hey @ArthurGW, this particular page is listing all the GO schemas, so this change it intended to clarify what the contents of the page are. In other places we refer to "Evo schemas", yep!